Source/JavaScriptCore/heap/IsoSubspace.cpp

```diff
@@ -80,7 +84,7 @@ void IsoSubspace::didBeginSweepingToFreeList(MarkedBlock::Handle* block)
         });
 }
 
-void* IsoSubspace::tryAllocateFromLowerTier()
+void* IsoSubspace::tryAllocateFromPreciseTier(size_t size)
```

Comment:
This is using `PreciseTier`, but it can be allocated from PreciseOnly allocation.

Reply:
Renamed to `tryAllocatePrecise`